packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -426,7 +433,13 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
       if (this.refs.selectionZone) {
         this.refs.selectionZone.ignoreNextFocus();
       }
-      this._async.setTimeout(() => row.focus(), 0);
+      this._async.setTimeout(() => {
+        const rowElement = ReactDOM.findDOMNode(row);
```

Comment:
this seems bad here because you're reaching into the row; if you ask the row to focus, the focusZone should pick up on that and set the focused element to that one.